src/main/resources/templates/cypher/form.html

```diff
@@ -52,6 +52,10 @@ <h2><span th:text="${heading}">Nadpis</span></h2>
                         <textarea id="placeDescription" class="form-control" th:field="*{placeDescription}"/>
                         <p class="error" th:if="${#fields.hasErrors('placeDescription')}" th:errors="*{placeDescription}">Place Description error.</p>
                     </div>
+                    <div class="form-group">
+                        <label for="organizer">Jméno organizátora</label>
```

Comment:
Nechci být otravný, ale možná bych dal popisel něco jako: Organizátor na stanovišti, nebo Přirazený organizátor...
Co na to ostatní?